command/pr.go

```diff
@@ -1,9 +1,11 @@
 package command
 
 import (
+	"bufio" // used to input comment
 	"errors"
 	"fmt"
 	"io"
+	"os" // used to input comment
```

Comment:
It's ok to leave these uncommented

```suggestion
	"os"
```